mirrord-agent/src/main.rs

```diff
@@ -288,15 +298,15 @@ async fn start_agent() -> Result<(), AgentError> {
                 match sniffer_output {
                     Some(sniffer_output) => {
                         match sniffer_output {
-                            SnifferOutput::NewTCPConnection(new_connection) => {
-                                debug!("SnifferOutput::NewTCPConnection -> connection {:#?}", new_connection);
+                            SnifferOutput::NewTcpConnection(new_connection) => {
+                                debug!("SnifferOutput::NewTcpConnection -> connection {:#?}", new_connection);
                                 let peer_ids = state.port_subscriptions.get_topic_subscribers(new_connection.destination_port);
 
                                 for peer_id in peer_ids {
                                     state.connections_subscriptions.subscribe(peer_id, new_connection.connection_id);
 
                                     if let Some(peer) = state.peers.get(&peer_id) {
-                                        match peer.channel.send(DaemonMessage::NewTCPConnection(new_connection.clone())).await {
+                                        match peer.channel.send(DaemonMessage::Tcp(DaemonTcp::NewConnection(new_connection.clone()))).await {
```

Comment:
`if let Err(fail) = ...` if we don't care about the `Ok` portion.

Reply:
I'll take care of this in next pr